ddtrace/settings/config.py

```diff
@@ -334,9 +366,18 @@ def __init__(self):
         except (TypeError, ValueError):
             pass
 
-        dd_trace_obfuscation_query_string_pattern = os.getenv(
-            "DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN", DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN_DEFAULT
-        )
+        if "DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN" in os.environ:
+            deprecate(
+                "`DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN` is deprecated "
+                "and will be removed in the next major version.",
+                message="use `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP` instead",
+                removal_version="2.0.0",
+            )
+            dd_trace_obfuscation_query_string_pattern = os.getenv("DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN")
+        else:
+            dd_trace_obfuscation_query_string_pattern = os.getenv(
```

Comment:
Non-blocking: Rename this internal attribute to the new name

Reply:
done